first-task/src/components/ModalAlbum/Album.jsx

```diff
@@ -0,0 +1,54 @@
+import React, { useContext, useState } from "react";
+import { AlbumContext } from "./AlbumContext";
+import AlbumItems from "./AlbumItems";
+import { Modal, Button } from "react-bootstrap";
+import AddForm from "./AddForm";
+
+const Album = () => {
+
+    const {albums} = useContext(AlbumContext);
+    const [show, setShow] = useState(false);
+
+    const showWindow = () => setShow(true);
+    const close = () => setShow(false);
```

Comment:
you need to use `useCallback` hook here

Reply:
also you can write one function `const toggleWindoe = useCallback(() => setSwoh(prevState => !prevState))`